site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -348,6 +348,37 @@ Workloads as possible, preferring Workloads with these characteristics:
 - Workloads with the lowest priority.
 - Workloads that have been admitted more recently.
 
+## FlavorFungibility
+
+When there is not enough nominal quota of resources in a ResourceFlavor, the incoming workload can borrow or preempt in ClusterQueue or Cohort. 
+
+By default the incoming workload will stop trying the next flavor if it can get enough resource by borrowing. 
```

Comment:
I think we should clarify what is `the next flavor`.

Reply:
Yeah, we can remind readers that kueue evaluates the flavors in a ClusterQueue in order.

Reply:
But generally, we introduce the possible options first, followed by what's the default.

Reply:
agree.

Reply:
Sure, I will update this